src/Commands/MysqlSlowCommand.php

```diff
@@ -160,7 +172,7 @@ private function ParseMysqlSlowLog($dir, $options)
      */
     private function DefineSiteEnv($site_env)
     {
-        list($this->site, $this->environment) = $this->getSiteEnv($site_env);
+        [$this->site, $this->environment] = $this->getSiteEnv($site_env);
```

Comment:
TIL this works all the way back to PHP 7.1.

Reply:
TIL multiple return values works in PHP at all